docs/contributing-analyzer.md

```diff
@@ -154,11 +159,8 @@ After the debug session, remove the `Debugger.Launch()` line.
   * Follow the [Code Style Configuration for Intellij](https://github.com/SonarSource/sonar-developer-toolset#code-style-configuration-for-intellij) instructions
   * Open the root folder of the repo
   * Make sure the `its`, `sonar-csharp-plugin`, `sonar-dotnet-shared-library`, and `sonar-vbnet-plugin` folders are are imported as Maven modules (indicated by a blue square). Search for `pom.xml` in the folders and make it a maven project if not.
-* Add the following environment variables (*user* scope)
-  * **ARTIFACTORY_URL** https://repox.jfrog.io/repox
-  * **ARTIFACTORY_USER** your repox.jfrog username (see e.g. orchestrator.properties)
-  * **ARTIFACTORY_PASSWORD** the api key for repox.jfrog (see e.g. orchestrator.properties)
 * Create `settings.xml` in the `%USERPROFILE%\.m2` directory. A template can be found in the [Developer box section in the extranet](https://xtranet-sonarsource.atlassian.net/wiki/spaces/DEV/pages/776711/Developer+Box#Maven-Settings). Change the username and password settings with the values from the environment variables above.
+* Make sure the orchestrator is configured [see here](https://xtranet-sonarsource.atlassian.net/wiki/spaces/DEV/pages/776679/Integration+Tests#Configuration).
```

Comment:
Why is this here? The information is redundant to the `ORCHESTRATOR_CONFIG_URL` environment variable
```suggestion
```

Reply:
True - and internals have no other choice than developing with an IDE.